### PR TITLE
OnclickMove Event for carousel

### PR DESCRIPTION
**Pull request by jawadakram20**

### CODE REVIEW

**Comment by amio** [comment]
Thanks for you PR!

Did you mean to let user slide carousel with a single click instead of dragging?
It seems works fine when clicking, but breaks dragging.
Dragging behavior becomes wired. Could you fix it?

And maybe it's better to add a switch for this feature?

File: carousel.js [hunk 1]
```diff
@@ -0,0 +1,381 @@
+'use strict';
```
Comment (amio) [inline carousel.js]: `carousel.js` and `indicator-dots.js` shouldn't be included, they are only built for release to npm.
I should refine the build process, avoid generate temporary files in root dir. 🤔

File: src/carousel.js [hunk 2]
```diff
@@ -80,6 +81,18 @@ class Carousel extends React.Component {
     this.refs.wrapper.addEventListener('mouseup', this.onTouchEnd)
   }
 
+  onClickMove (e) {
+    var _this2 = this;
```
Comment (amio) [inline src/carousel.js]: This can't pass eslint check, please remove those semicolons.

**Comment by jawadakram20** [comment]
@amio Can you please update readme.md with command to run this locally

**Pull request closed without merging**